src/components/Sidebar/chat/history.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { TbClockHour4 } from 'react-icons/tb'
+import { RiArrowDownSLine } from 'react-icons/ri'
+import { ChatHistory } from '../../../hooks/useOpenAI'
+interface HistoryProps {
+    chatHistory: ChatHistory[]
+    setCurrentId: (chatID: string) => void
+    currentUrl: string
+    currentId: string
+}
+export const History = ({ chatHistory, setCurrentId, currentUrl, currentId }: HistoryProps) => {
```

Comment:
I believe we should use this https://www.radix-ui.com/docs/primitives/components/dropdown-menu with it we can get keyboard accessibility free to move easily between histories

Reply:
okay!